src/app/myTraining/myTrainingCalendar/MyTrainingCalendar.tsx

```diff
@@ -98,31 +98,33 @@ const MyTrainingCalendar = () => {
                 <div className={'item-workout-container'}>
                   {workouts[`${date.format('YYYY-MM-DD')}`] &&
                     workouts[`${date.format('YYYY-MM-DD')}`].map((workout) => (
-                      <Tooltip
-                        disableInteractive
-                        title={
-                          <>
-                            {workout.name} <br /> {workout.exercises.length}
-                            {workout.exercises.length === 1 ? ' exercise' : ' exercises'}
-                          </>
-                        }
-                        slotProps={{
-                          popper: {
-                            modifiers: [
-                              {
-                                name: 'offset',
-                                options: {
-                                  offset: [0, -8],
+                      <Fragment key={workout.id}>
+                        <Tooltip
+                          disableInteractive
+                          title={
+                            <>
+                              {workout.name} <br /> {workout.exercises.length}
+                              {workout.exercises.length === 1 ? ' exercise' : ' exercises'}
+                            </>
+                          }
+                          slotProps={{
+                            popper: {
+                              modifiers: [
+                                {
+                                  name: 'offset',
+                                  options: {
+                                    offset: [0, -8],
+                                  },
                                 },
-                              },
-                            ],
-                          },
-                        }}
-                      >
-                        <div onClick={() => navigateToWorkout(workout.id)} key={workout.id} className={'item-workout'}>
-                          {workout.name}
-                        </div>
-                      </Tooltip>
+                              ],
```

Comment:
It's only identation change, because of adding fragment it had to be 1 identation more